anypoint-mq/4.x/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,125 @@
+= Anypoint MQ Connector 4.x — Mule 4
+
+Support Category: Select
+
+The Anypoint MQ connector provides publish-subscribe messaging to Mule apps using the Anypoint MQ cloud service.
+
+Anypoint MQ supports a wide range of messaging use cases, such as enterprise messaging patterns, asynchronous communications between microservices, and IoT. Anypoint MQ also has a REST API for publishing or consuming messages using frameworks such as node.js, Java, Go, and DevOps bash scripts.
+
+== Requirements
+
+To use this connector, you should be familiar with Anypoint Studio, Anypoint Connectors, and application flows. The Anypoint MQ connector is free, but connecting to the service in Anypoint Platform requires a license.
+
+=== Compatibility
+
+[%header%autowidth.spread]
+|===
+|Software |Version
+|Mule runtime engine |4.1.1 and later
+|Anypoint Studio |7 and later
+|===
+
+This documentation applies to Anypoint MQ connector 4.x only. To migrate from version 3.x to the version 4.x, see the xref:anypoint-mq-connector-upgrade-migrate.adoc[].
+
+[[install-mq-connector]]
+== Install Anypoint MQ Connector
+
+You can use this connector with apps written in Anypoint Studio, standalone apps, or apps written in XML.
+
+In Anypoint Studio:
+
+. Create or open a Mule project.
+. Click the *Exchange* icon in the Studio task bar.
+. Enter your Anypoint Platform username and password and click *Sign in*.
+. Select *All assets* and search for "MQ".
+. Select "Anypoint MQ Connector - Mule 4", and click *Add to project*.
```

Comment:
Here, I would use asterisks for Anypoint MQ Connector - Mule 4 since this is in the GUI.